app/models/shift.rb

```diff
@@ -19,6 +19,35 @@ class Shift < ApplicationRecord
   before_destroy :notify_volunteers_about_destroy, prepend: true
   before_update  :notify_volunteers_about_update, prepend: true
 
+  def self.filtered_for_ngo(ngo, filters)
+    the_scope, order_by = filters
+    # unscoped is necessary because default_scope (deprecated?) is
+    # messing up the order by
+    shifts =
+      unscoped
```

Comment:
`default_scope` is stomping on my weird `order by`. I will have to add the null and deleted_at explicitly. Thanks for the heads up.